src/Components/MovieDetails/MovieDetails.css

```diff
@@ -0,0 +1,169 @@
+/* * {
```

Comment:
Love all these changes, but they don't seem to be related to refactoring API calls - I might have made these their own PR, to be honest